lib/active_merchant/billing/gateways/pay_gate_xml.rb

```diff
@@ -192,15 +199,16 @@ def build_request(action, options={})
         xml.instruct!
 
         xml.tag! 'protocol', :ver => API_VERSION, :pgid => (test? ? TEST_ID : @options[:login]), :pwd => @options[:password] do |protocol|
+          money       = options.delete(:money)
+          authorization   = options.delete(:authorization)
+          creditcard  = options.delete(:creditcard)
```

Comment:
I think you intended to line up the `=` here? Sorry to be so picky about whitespace. ;)